src/Microsoft.Docs.Template/Models/FileMetadata.cs

```diff
@@ -27,5 +27,7 @@ public class FileMetadata
 
         [JsonExtensionData]
         public JObject ExtensionData { get; set; }
+
+        public bool ShouldSerializeMonikerRange() => false;
```

Comment:
modify a test case to assert `monikerRange` is not in output

Reply:
sure